views/default/tld_rows.pdt

```diff
@@ -0,0 +1,39 @@
+<?php
+$i = 0;
+foreach ($tlds as $tld => $pricing) {
+    foreach ($currencies as $currency) {
+        ?>
+        <tr class="price_row<?php echo $this->Html->concat(' ', ($i % 2 == 1) ? ' odd_row' : '', ''); ?> tld_<?php echo $this->Html->safe(trim($tld, '.')); ?>">
+            <td class="medium">
+                <strong><?php echo $this->Html->safe($tld); ?></strong>
```

Comment:
Instead of \<?php echo $this->Html->safe(...); ?> you can use  \<?php $this->Html->_(...)?>